t/tomcat/build_info.json

```diff
@@ -0,0 +1,11 @@
+{
+    "package_name" : "tomcat",
+    "github_url": "https://github.com/apache/tomcat.git",
+    "version": "11.0.0-M3",
+    "default_branch": "main",
+    "package_dir": "t/tomcat/",
+    "build_script": "tomcat_ubi_8.5.sh",
+    "validate_build_script": true,
+    "use_non_root_user": true,
```

Comment:
please make this "use_non_root_user": true, ---> "use_non_root_user": false.

Here `use_non_root_user: true -- normal user without sudo permission`
`use_non_root_user:false -- normal user with sudo permission`